### DISCUSSION TITLE
[米家同好群]QQ群招募

### DISCUSSION THREAD

**Discussion by shidijun** [share121/inter-knot#4873] [绝区零]
<div>
<div align="center"><h1>[米家同好群]招募</h1></div>
<p>
向各位米游玩家们致以敬意。
</p>
米游玩家一直希望有个好的圈内氛围已是游戏圈人尽皆知的事实。怎奈天违人愿，总有一些米哈游黑粉（以下简称“米黑”）打着以各种各样、虚实相掺的旗号，刻意扰乱米圈氛围和抹黑米游及米游玩家对外形象。对此，我们希望米游玩家团结起来，而不是被不良舆论影响，或是“胳膊肘往外拐”。
<p>
以召集米游玩家、创建一个良好氛围的圈子供各位米游玩家进行友好讨论的目的，我们一众志同道合的玩家创建了一个微信群聊，恭迎各位米游玩家通过扫描群聊二维码加入（群聊二维码见“附1”）或是添加“史蒂君ovo”好友加入（见“附2”）（或可添加附3或附4），无关人士还望止步。
</p>
<p>
本群聊为正经、合法群聊，望加入者遵守群公告中的各项规定，违者后果自负；无关人士一概不予入群，无关人士入群者核实后踢出群聊。
</p>
<p>
恭迎各位有意的米游玩家加入。
</p> 
<p>
</p>
<p>注：此群为Q群分群</p>
<p></p>
<h2>群主QQ号：3834837974</h2>            
                                                         <div align="right"><p>米家同好群 （writer:by史蒂君）</p>                                 
                                                                        <p>2024.9.14</p></div>

<h2>附1</h2>
<img src="https://github.com/user-attachments/assets/b615d7f5-6535-4e93-9c57-f89f42e1c769" alt="1">
<h2>附2</h2>
<img src="https://github.com/user-attachments/assets/b59ec74f-9ba8-43a1-a075-9f901b4e3a4d" alt="2">
<h2>附3</h2>
<img src="https://github.com/user-attachments/assets/d9cc7391-b530-4455-a1b3-07e273ca038e" alt="3">
<h2>附4</h2>
<img src="https://github.com/user-attachments/assets/951d7159-3519-4981-91a5-9e9861b6899b" alt="4">

**Comment by shidijun** [comment]
没人我自己顶<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by mztxy229** [comment]
顶你一下<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by xiarideling** [comment]
蹭蹭（那个表情的脚本怎么又用不起来了）

**Comment by shidijun** [comment]
又没热度？我顶顶顶顶顶顶顶顶顶<img src="https://img-static.mihoyo.com/communityweb/upload/1d9f61de2dca25089a49696a3d5de855.png" alt="打你" width="75" >

**Comment by shidijun** [comment]
顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶顶<img src="https://bbs-static.miyoushe.com/static/2024/03/21/86aed25342a127321aa7a469f111d52d_2959126012583825986.png" alt="丽都漫步-铃" width="75" >

**Comment by shidijun** [comment]
顶......<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by Herrscher-of-Origin** [comment]
散厨和散黑哪个能进？

**Reply by xiarideling** [reply]
你是哪一种呢<img src="https://img-static.mihoyo.com/communityweb/upload/d4d609d5af6bc85a0c3bba6b4b59fffa.png" alt="米游姬-暗中观察" width="75" >

**Reply by Herrscher-of-Origin** [reply]
> 麻烦您看一下正文 我想群友们可能更欢迎散厨
> 
> 找茬者还是别进的好

为什么说散黑就是找茬者，散黑又不是米黑，难道玩崩三厌恶奥托的也是米黑吗，我想只要没那种刻意提起这个话题引得两边人互战，应该也人有没错吧，怎么能就因为某玩家喜欢和讨厌的原神角色和别人不同，就说对方是米黑是找茬者呢？找茬者分的是人，不是标签

**Reply by shidijun** [reply]
抱歉 我为自己不当的言行深感抱歉 另外我说的“找茬者”也不是针对散黑 而是针对米黑 您可能理解错误了吧 并不是针对您

**Reply by linzhiyao123** [reply]
> > 麻烦您看一下正文 我想群友们可能更欢迎散厨
> > 找茬者还是别进的好
> 
> 为什么说散黑就是找茬者，散黑又不是米黑，难道玩崩三厌恶奥托的也是米黑吗，我想只要没那种刻意提起这个话题引得两边人互战，应该也人有没错吧，怎么能就因为某玩家喜欢和讨厌的原神角色和别人不同，就说对方是米黑是找茬者呢？找茬者分的是人，不是标签

那你为什么要问这种问题呢？这里是米家同好群招募，米黑止步，而散厨散黑和米黑没什么关系，也和能不能进群没关系，我看你这不是挺明白的吗？楼主确实如他所说有些言行不当，但是我看你这问题有点像引战的，没有别的意思，只是觉得你的问题问的就不恰当

**Reply by mztxy229** [reply]
散黑能进，散黑当然能进，前提是不要在群友们友善讨论涉及散兵的话题时冒出来说一句“散兵是xxx”。为什么讨厌散黑，因为现在散黑这个词前面带了一个隐形的“无脑”，如果你是理智的，不会没事找事，我们当然欢迎，但是我看你这个回复估计就是那种没事找事的人了

**Reply by mztxy229** [reply]
无脑散黑和无脑散厨没区别的招人嫌，所以你这个问题就不应该存在，我们群不欢迎过激的玩家

**Comment by shidijun** [comment]
顶

**Comment by shidijun** [comment]
顶一下

**Comment by shidijun** [comment]
顶顶顶<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by anewperson001** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/10/10/6fe05fae170b754c2c1afe3f1f6c847c_2945463426120653954.png" alt="卡琳_wink" width="75" >希希娜可到此一游

**Reply by shidijun** [reply]
抓到啦<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by shidijun** [comment]
顶顶顶<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by shidijun** [comment]
老子顶

**Comment by shidijun** [comment]
我又活了哈哈哈哈！顶顶顶！！！！！

**Comment by shidijun** [comment]
顶顶顶

**Comment by shidijun** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/3b8cf75f15864abfc9eeb1b9316d9224.png" alt="胡桃-卖萌" width="75" >顶

**Comment by shidijun** [comment]
顶顶顶

**Comment by shidijun** [comment]
。
